enterprise/models.py

```diff
@@ -821,13 +822,15 @@ def get_link_by_email(self, user_email, enterprise_customer):
 
         return None
 
-    def link_user(self, enterprise_customer, user_email):
+    def link_user(self, enterprise_customer, user_email, **kwargs):
         """
         Link user email to Enterprise Customer.
 
         If :class:`django.contrib.auth.models.User` instance with specified email does not exist,
         :class:`.PendingEnterpriseCustomerUser` instance is created instead.
         """
+        defaults = {'active': True}
+        defaults.update(kwargs)
```

Comment:
This was a change to support testing that I'm now realizing is no longer needed, will delete.